src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/ProductAliasChip.tsx

```diff
@@ -52,7 +52,7 @@ export const ProductAliasChip = ({color, text, selected, onPress}: Props) => {
 
 const useStyles = StyleSheet.createThemeHook((theme) => ({
   container: {
-    borderRadius: theme.border.radius.circle * 2,
```

Comment:
The value that previously was `circle` has been renamed to `large`, and `circle` has been set to `999`